dask_sql/physical/rel/logical/aggregate.py

```diff
@@ -131,6 +131,23 @@ class DaskAggregatePlugin(BaseRelPlugin):
             )
         ),
         "avg": AggregationSpecification("mean", AggregationOnPandas("mean")),
+        "stddev": AggregationSpecification("std", AggregationOnPandas("std")),
+        "stddevsamp": AggregationSpecification("std", AggregationOnPandas("std")),
+        "stddevpop": AggregationSpecification(
+            dd.Aggregation(
```

Comment:
The logic generally looks good. 

Could you add a few tests for these aggregations. (preferable some that test on the gpu as well)